patches/instance.yml

```diff
@@ -1,22 +1,24 @@
 Change:
   Instance:
-    # UniqueId properties might be random everytime Studio saves a place file
-    # and don't have a use right now outside of packages, which Rojo doesn't
-    # account for anyway. They generate diff noise, so we shouldn't serialize
-    # them until we have to.
+    # UniqueId is randomized per Studio load, but that's not a useful default.
+    # Rather than getting the default from Studio, we manually specify it here
+    # instead.
     UniqueId:
-      Serialization:
-        Type: DoesNotSerialize
+      DefaultValue:
+        UniqueId: "00000000000000000000000000000000"
     HistoryId:
-      Serialization:
-        Type: DoesNotSerialize
+      DefaultValue:
+        UniqueId: "00000000000000000000000000000000"
```

Comment:
Might be a good idea to leave a comment explaining why these values were chosen (likewise for ScriptGuid)